.travis.yml

```diff
@@ -7,11 +7,27 @@ arch:
   - s390x
   - ppc64le
   - amd64
+env:
+  - NIGHTLY=true
+  - NIGHTLY=false
```

Comment:
Why set NIGHTLY twice here?

Reply:
It's to specify all values of NIGHTLY that needs to be tested. So the combination of arch and env expands to:
```
s390x-true
s390x-false
ppc64le-true
ppc64le-false
...
```
resulting in 6 jobs.